deploy/kubernetes/examples/demo-storageclass-using-secret-from-configuration.yaml

```diff
@@ -0,0 +1,25 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: demo-storageclass
+provisioner: block.csi.ibm.com
+volumeBindingMode: WaitForFirstConsumer  # Optional. Required for Topology Awareness support.
+parameters:
+  # a pool value MUST be provided for all systems.

```

Comment:
IMO this comment combined with `pool: demo-pool                # Optional.` is confusing.
I think we should leave `pool` without a comment, as was done for the regular storageclass

Reply:
I meant that this comment should be removed as well

Reply:
non `Optional` == required?

Reply:
done